src/core.c

```diff
@@ -606,10 +605,10 @@ int connect_proxy_chain(int sock, ip_type target_ip,
 
 		case ROUND_ROBIN_TYPE:
 			alive_count = calc_alive(pd, proxy_count);
-			curr_pos = offset = proxychains_proxy_offset;
+			offset = proxychains_proxy_offset;
 			if(alive_count < max_chain)
 				goto error_more;
-                        PDEBUG("1:rr_offset = %d, curr_pos = %d\n", offset, curr_pos);
+				PDEBUG("1:rr_offset = %d\n", offset);
```

Comment:
the added indentation here is wrong, it makes it look as if the statement belongs to the if(...) above

Reply:
Fixed